docs/pages/zero-trust-access/management/operations/breakglass-access.mdx

```diff
@@ -0,0 +1,433 @@
+
+(!docs/pages/includes/edition-prereqs-tabs.mdx!)
+
+## Step 1/6. Configure sshd to trust the Teleport CA
+
+For break-glass access, the OpenSSH server must be configured to trust client certificates issued by the Teleport `user` Certificate Authority (CA).
+
+1. Export the public key of the Teleport `user` CA:
+
+**This step is completed on the Teleport Agent machine.**
+
+On the Teleport Agent machine, run the following command. Replace `teleport.example.com` with the address of your Teleport Proxy Service:
+
+```bash
+export KEY=$(curl 'https://<Var name="teleport.example.com" />/webapi/auth/export?type=user' | sed "s/cert-authority\ //")
+```
+
+2. Make the public key accessible to `sshd`:
+
+```bash
+echo "$KEY" | sudo tee /etc/ssh/teleport_user_ca.pub
+echo "TrustedUserCAKeys /etc/ssh/teleport_user_ca.pub" | sudo tee -a /etc/ssh/sshd_config
+```
+
+3. Check the syntax of the sshd config file (you should see no errors printed):
+
+```bash
+sudo sshd -t
+```
+
+4. Restart the sshd service:
+
+```bash
+# This unit may also be called ssh.service depending on your distribution
+# Check with 'systemctl list-unit-files | grep ssh | grep service'
+sudo systemctl restart sshd
+```
+
+Now, `sshd` will trust users who present a Teleport-issued SSH certificate.
+
+<Admonition type="important">
+You must complete Steps 4 and 5 below to ensure only the `breakglass` user is allowed to log in with a valid Teleport certificate
+and is configured with proper access restrictions, in order to prevent user switching once logged in.
+</Admonition>
+
+## Step 2/6. Create a `breakglass` user on the Teleport Agent server
+
+**This step is completed on the Teleport Agent machine.**
+
+Log into the Teleport Agent server and run the following commands.
+
+1. Create the `breakglass` user:
+
+```bash
+sudo adduser --disabled-password breakglass
+```
+
+Set a password for `breakglass` when prompted. We recommend the use of a long, random string (e.g. the output of `pwgen -s 64 1`)
+as this password will never be used for access.
+
+<Admonition type="note">
+As a general security principle, it is highly recommended to limit `root` permissions through `sudo`. Because `breakglass` will need
+ability to gain elevated access to the system to troubleshoot and recover access, however, any use of break-glass access should always be audited
+via `sshd` access logs.
+</Admonition>
+
+2: Edit the sudoers file using `visudo`:
+
+```bash
+sudo visudo
+```
+
+3. Grant limited sudo permissions:
+
+Here is an example of how to restrict user switching and allow the `breakglass` user only to specific privileged commands
+(e.g. `cat`, `ls`, `journalctl`, `systemctl`, and `vim`):
+
+```bash
+breakglass ALL=(ALL) NOPASSWD: !ALL
+breakglass ALL=(ALL) NOPASSWD: /usr/bin/cat, /usr/bin/ls, /usr/bin/journalctl, /usr/bin/systemctl
+breakglass ALL=(ALL) NOPASSWD: NOEXEC: /usr/bin/vim
+breakglass ALL=(ALL) NOPASSWD: !/usr/bin/su, !/usr/bin/sudo
+```
```

Comment:
Can we just get rid of this? We should not pretend it's possible to allow unrestricted `systemctl` executions and meaningfully restrict what's going to happen afterwards.